[2025-03-10T16:21:39.314+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T16:21:39.338+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T16:21:39.344+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T16:21:39.344+0000] {taskinstance.py:2867} INFO - Starting attempt 15 of 16
[2025-03-10T16:21:39.353+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run> on 2025-03-10 14:20:39.848994+00:00
[2025-03-10T16:21:39.358+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2791) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-10T16:21:39.359+0000] {standard_task_runner.py:72} INFO - Started process 2792 to run task
[2025-03-10T16:21:39.358+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xkcd_daily', 'dbt_run', 'manual__2025-03-10T14:20:39.848994+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/XKCD_daily/dag.py', '--cfg-path', '/tmp/tmpk_do8iyx']
[2025-03-10T16:21:39.359+0000] {standard_task_runner.py:105} INFO - Job 162: Subtask dbt_run
[2025-03-10T16:21:39.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T16:21:39.416+0000] {task_command.py:467} INFO - Running <TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [running]> on host 157a64d0872f
[2025-03-10T16:21:39.488+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xkcd_daily' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T14:20:39.848994+00:00' AIRFLOW_CTX_TRY_NUMBER='15' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T14:20:39.848994+00:00'
[2025-03-10T16:21:39.488+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T16:21:39.498+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-10T16:21:39.498+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /dbt/transformations && dbt run --project-dir /dbt/transformations --profiles-dir /dbt']
[2025-03-10T16:21:39.504+0000] {subprocess.py:99} INFO - Output:
[2025-03-10T16:21:41.049+0000] {subprocess.py:106} INFO - [0m16:21:41  Running with dbt=1.9.3
[2025-03-10T16:21:41.264+0000] {subprocess.py:106} INFO - [0m16:21:41  Registered adapter: postgres=1.9.0
[2025-03-10T16:21:41.687+0000] {subprocess.py:106} INFO - [0m16:21:41  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-10T16:21:41.687+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-03-10T16:21:41.688+0000] {subprocess.py:106} INFO - - models.transformations.example
[2025-03-10T16:21:41.797+0000] {subprocess.py:106} INFO - [0m16:21:41  Found 5 models, 1 source, 433 macros
[2025-03-10T16:21:41.799+0000] {subprocess.py:106} INFO - [0m16:21:41
[2025-03-10T16:21:41.799+0000] {subprocess.py:106} INFO - [0m16:21:41  Concurrency: 1 threads (target='prod')
[2025-03-10T16:21:41.799+0000] {subprocess.py:106} INFO - [0m16:21:41
[2025-03-10T16:21:41.904+0000] {subprocess.py:106} INFO - [0m16:21:41  1 of 5 START sql table model dbt_dwh.dim_comics ................................ [RUN]
[2025-03-10T16:21:42.039+0000] {subprocess.py:106} INFO - [0m16:21:42  1 of 5 OK created sql table model dbt_dwh.dim_comics ........................... [[32mSELECT 561[0m in 0.13s]
[2025-03-10T16:21:42.045+0000] {subprocess.py:106} INFO - [0m16:21:42  2 of 5 START sql table model dbt_dwh.fact_cost ................................. [RUN]
[2025-03-10T16:21:42.112+0000] {subprocess.py:106} INFO - [0m16:21:42  2 of 5 OK created sql table model dbt_dwh.fact_cost ............................ [[32mSELECT 561[0m in 0.06s]
[2025-03-10T16:21:42.113+0000] {subprocess.py:106} INFO - [0m16:21:42  3 of 5 START sql table model dbt_dwh.fact_review ............................... [RUN]
[2025-03-10T16:21:42.213+0000] {subprocess.py:106} INFO - [0m16:21:42  3 of 5 OK created sql table model dbt_dwh.fact_review .......................... [[32mSELECT 561[0m in 0.10s]
[2025-03-10T16:21:42.215+0000] {subprocess.py:106} INFO - [0m16:21:42  4 of 5 START sql table model dbt_dwh.fact_views ................................ [RUN]
[2025-03-10T16:21:42.282+0000] {subprocess.py:106} INFO - [0m16:21:42  4 of 5 OK created sql table model dbt_dwh.fact_views ........................... [[32mSELECT 561[0m in 0.07s]
[2025-03-10T16:21:42.284+0000] {subprocess.py:106} INFO - [0m16:21:42  5 of 5 START sql view model dbt_dwh.dm_comic_overview .......................... [RUN]
[2025-03-10T16:21:42.352+0000] {subprocess.py:106} INFO - [0m16:21:42  5 of 5 OK created sql view model dbt_dwh.dm_comic_overview ..................... [[32mCREATE VIEW[0m in 0.07s]
[2025-03-10T16:21:42.385+0000] {subprocess.py:106} INFO - [0m16:21:42
[2025-03-10T16:21:42.386+0000] {subprocess.py:106} INFO - [0m16:21:42  Finished running 4 table models, 1 view model in 0 hours 0 minutes and 0.59 seconds (0.59s).
[2025-03-10T16:21:42.435+0000] {subprocess.py:106} INFO - [0m16:21:42
[2025-03-10T16:21:42.435+0000] {subprocess.py:106} INFO - [0m16:21:42  [32mCompleted successfully[0m
[2025-03-10T16:21:42.436+0000] {subprocess.py:106} INFO - [0m16:21:42
[2025-03-10T16:21:42.436+0000] {subprocess.py:106} INFO - [0m16:21:42  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-03-10T16:21:43.112+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-10T16:21:43.148+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T16:21:43.149+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=xkcd_daily, task_id=dbt_run, run_id=manual__2025-03-10T14:20:39.848994+00:00, execution_date=20250310T142039, start_date=20250310T162139, end_date=20250310T162143
[2025-03-10T16:21:43.223+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T16:21:43.242+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10T16:21:43.245+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
